{"created": 1652300775.125877, "duration": 0.11057186126708984, "exitcode": 1, "root": "C:\\Users\\Daniel\\OneDrive\\\u0420\u0430\u0431\u043e\u0447\u0438\u0439 \u0441\u0442\u043e\u043b\\game_agent_copy\\game_agent\\course\\tests\\python", "environment": {"Python": "3.10.3", "Platform": "Windows-10-10.0.19044-SP0", "Packages": {"pytest": "7.1.2", "py": "1.11.0", "pluggy": "1.0.0"}, "Plugins": {"json-report": "1.5.0", "metadata": "2.0.1"}}, "summary": {"passed": 2, "failed": 1, "total": 3, "collected": 3}, "collectors": [{"nodeid": "", "outcome": "passed", "result": [{"nodeid": "include.py", "type": "Module"}]}, {"nodeid": "include.py", "outcome": "passed", "result": [{"nodeid": "include.py::test_upper", "type": "Function", "lineno": 10}, {"nodeid": "include.py::test_isupper", "type": "Function", "lineno": 13}, {"nodeid": "include.py::test_failed_upper", "type": "Function", "lineno": 16}]}], "tests": [{"nodeid": "include.py::test_upper", "lineno": 10, "outcome": "passed", "keywords": ["test_upper", "include.py", "python"], "setup": {"duration": 0.0007917000002635177, "outcome": "passed"}, "call": {"duration": 0.00026619999698596075, "outcome": "passed"}, "teardown": {"duration": 0.0002824000002874527, "outcome": "passed"}}, {"nodeid": "include.py::test_isupper", "lineno": 13, "outcome": "passed", "keywords": ["test_isupper", "include.py", "python"], "setup": {"duration": 0.0004288000018277671, "outcome": "passed"}, "call": {"duration": 0.00035959999877377413, "outcome": "passed"}, "teardown": {"duration": 0.0003842000005533919, "outcome": "passed"}}, {"nodeid": "include.py::test_failed_upper", "lineno": 16, "outcome": "failed", "keywords": ["test_failed_upper", "include.py", "python"], "setup": {"duration": 0.0009739999986777548, "outcome": "passed"}, "call": {"duration": 0.005736800001614029, "outcome": "failed", "crash": {"path": "C:\\Users\\Daniel\\OneDrive\\\u0420\u0430\u0431\u043e\u0447\u0438\u0439 \u0441\u0442\u043e\u043b\\game_agent_copy\\game_agent\\course\\tests\\python\\include.py", "lineno": 18, "message": "AssertionError: assert 'FOO' == 'FOo'\n  - FOo\n  + FOO"}, "traceback": [{"path": "include.py", "lineno": 18, "message": "AssertionError"}], "longrepr": "def test_failed_upper():\n>       assert 'foo'.upper() == 'FOo'\nE       AssertionError: assert 'FOO' == 'FOo'\nE         - FOo\nE         + FOO\n\ninclude.py:18: AssertionError"}, "teardown": {"duration": 0.0006829999983892776, "outcome": "passed"}}]}